src/types/license.types.ts

```diff
@@ -292,6 +293,7 @@ export const licenses: LicenseInfo[] = [
     description: i18n.t('licenses.description.cc0'),
     link: i18n.t('licenses.links.cc0'),
     logo: LicenseImages.cc0Logo,
+    i18nKey: 'cc0',
```

Comment:
Hvorfor legge inn en ny property som bare har i18n key her samtidig som objektet har 3 andre properties som er fulle tekster basert på key?
Burde være enten bare tekster eller bare keys ville jeg tenkt, for når jeg ser objektet her skjønner jeg ikke hva `i18nKey: 'cc0'` skal være uten å måtte lese videre i flere andre filer.
Om denne skulle hatt `i18nKey` skulle den vel også være den faktiske key-en (`licenses.additional_info.cc0`), og ikke bare siste delen av det? For ellers må man igjen lese videre alle plassser hvor denne brukes for å vite hvilken key som brukes og hva man skal endre